### PR TITLE
Add more 'NEXT_ACTIONS'

### PR DESCRIPTION
**Pull request by adammeyer**

### CODE REVIEW

**Comment by soberstadt** [comment]
Nice additions. Looks good.

**Comment by soberstadt** [comment]
I'm experiencing Call for Decision not making the new task subject 'Call for Decision'. Do you think this could have broken that.

File: app/assets/javascripts/angular/directives/taskFollowupDialog.js [hunk 1]
```diff
@@ -64,25 +64,25 @@ angular.module('mpdxApp')
...
-                       strContains(taskResult, 'Text Again')) {
+                       strContains(taskResult, 'Email') ||
+                       strContains(taskResult, 'Message') ||
+                       strContains(taskResult, 'Text')) {
 
                         //generic followup task type
                         var taskType;
                         var taskSubject;
-                        if(strContains(taskResult, 'Call Again')){
+                        if(strContains(taskResult, 'Call')){
```
Comment (soberstadt) [inline app/assets/javascripts/angular/directives/taskFollowupDialog.js]: Ah yes, this change. Can you explain why you did this?

Comment (adammeyer) [inline app/assets/javascripts/angular/directives/taskFollowupDialog.js]: You can now select 'Call' from any task type.  So ' Again' needed to be removed.  I fixed the issue here: https://github.com/CruGlobal/mpdx/pull/244  plus added some more changes requested by Stacie.

Comment (soberstadt) [inline app/assets/javascripts/angular/directives/taskFollowupDialog.js]: Gotcha. Sounds good.

**Pull request merged**